app/assets/stylesheets/gems.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the gems controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
If you're not going to add any CSS or JS right now I think it's safe to not add this files since they are only generated by Rails generators in case you really need them.